css/style.css

```diff
@@ -1,8 +1,44 @@
+
+
+
+   
+
+
+
 /**
  * Add your custom styles below
+
  * 
  * Remember: 
  * - Be organised, use comments and separate your styles into meaningful chunks
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+body
+{
+    Color: blue;
+    background-image: url("https://img.freepik.com/premium-photo/set-various-social-media-blocks-blue-painted-wall_23-2147841215.jpg?w=2000");
+    font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    font-size: medium;
+}
+.facebook{
```

Comment:
Can you think of a way to make a general class that controls the layout for all the articles on this page?

What does `.class` mean? How is it different from `#id` ?